src/api/app/controllers/webui/kiwi/images_controller.rb

```diff
@@ -24,18 +24,49 @@ def import_from_package
           end
         end
 
-        redirect_to kiwi_image_repositories_path(package.kiwi_image)
+        redirect_to kiwi_image_path(package.kiwi_image)
       end
 
       def show
+        @repositories = @image.repositories.order(:order)
+        @package = @image.package
+        @project = @package.project
+      end
+
+      def edit
+        @repositories = @image.repositories.order(:order)
+        @package = @image.package
+        @project = @package.project
+      end
+
+      def update
+        ::Kiwi::Image.transaction do
+          @image.update_attributes!(image_params)
+          @image.write_to_backend
+        end
+        redirect_to action: :show
+      rescue => e
```

Comment:
Please no generic exception handling! I know you just move the code around, but please fix this! We should only catch exceptions we expect to happen